var/spack/repos/builtin/packages/petsc/package.py

```diff
@@ -330,7 +331,8 @@ def install(self, spec, prefix):
 
         # PETSc depends on scalapack when '+mumps+mpi~int64' (see depends())
         # help PETSc pick up Scalapack from MKL
-        if spec.satisfies('+mumps+mpi~int64'):
+        if (spec.satisfies('+mumps+mpi~int64') or
+                spec.satisfies('@3.13:+mumps+mpi')):
```

Comment:
So this can also be updated to:
```
-        if (spec.satisfies('+mumps+mpi~int64') or
-                spec.satisfies('@3.13:+mumps+mpi')):
+        if (spec.satisfies('+mumps')):
```

Perhaps @adamjstewart can review these changes.